zabbix_cli/cli.py

```diff
@@ -4453,24 +4464,35 @@ def do_acknowledge_event(self, args):
                 print('--------------------------------------------------------')
                 event_ids = input('# EventIDs: ').strip()
                 ack_message = input('# Message[' + ack_message_default + ']:').strip()
+                close = input('# Close[' + ack_close_default + ']:').strip().lower()
                 print('--------------------------------------------------------')
 
             except EOFError:
                 print('\n--------------------------------------------------------')
                 print('\n[Aborted] Command interrupted by the user.\n')
                 return False
-
-        elif len(arg_list) == 2:
+        elif len(arg_list) >= 2:
```

Comment:
It's better to keep the `len(arg_list) == 2` case here and just set `close = ack_close_default` in it, and then do another `elif len(arg_list) == 3`; otherwise the `else` clause below is never reached if the user passes > 3 arguments.